pathfinder/tests/fixtures/core.py

```diff
@@ -1,42 +1,47 @@
 import random
 from typing import List
 from unittest.mock import Mock
-
 import pytest
-from eth_utils import to_checksum_address, encode_hex, keccak
+from coincurve import PrivateKey
+from eth_utils import to_checksum_address, remove_0x_prefix
 from raiden_libs.contracts import ContractManager
+from raiden_libs.utils import public_key_to_address
 from web3 import Web3
 from web3.contract import Contract
-
 from pathfinder.pathfinding_service import PathfindingService
+from pathfinder.tests.config import NUMBER_OF_CHANNELS
 from pathfinder.token_network import TokenNetwork
+from pathfinder.utils.types import Address
 
 
 @pytest.fixture
 def populate_token_networks(
         token_networks: List[TokenNetwork],
         token_network_contracts: List[Contract],
+        addresses: List[Address],
+        private_key_seeds: List[str],
 ) -> None:
-
+    random.seed(NUMBER_OF_CHANNELS)
     for token_network in token_networks:
         # some random magic to add channels
-        for channel_id in range(100):
-            seed1, seed2 = random.sample(range(20), 2)
-
-            p1 = to_checksum_address(encode_hex(keccak(seed1))[:42])
-            p2 = to_checksum_address(encode_hex(keccak(seed2))[:42])
-
+        for channel_id in range(NUMBER_OF_CHANNELS):
+            seed1, seed2 = random.sample(private_key_seeds, 2)
+            private_key1 = PrivateKey.from_hex(remove_0x_prefix(seed1))
+            private_key2 = PrivateKey.from_hex(remove_0x_prefix(seed2))
+            public_key1 = to_checksum_address(public_key_to_address(private_key1.public_key))
+            public_key2 = to_checksum_address(public_key_to_address(private_key2.public_key))
+            fee1 = str(abs(random.gauss(0.0002, 0.0001))).encode()
+            fee2 = str(abs(random.gauss(0.0002, 0.0001))).encode()
+            signature1 = private_key1.sign_recoverable(fee1)
+            signature2 = private_key2.sign_recoverable(fee2)
             token_network.handle_channel_opened_event(
                 channel_id,
-                p1,
-                p2
+                public_key1,
+                public_key2
             )
 
-        # deposit to channels
-        for channel_id in range(100):
-            deposit1 = random.randint(0, 1000)
-            deposit2 = random.randint(0, 1000)
-
+            # deposit to channels
+            deposit1, deposit2 = random.sample(range(1000), 2)
```

Comment:
We might want to make the max deposit configurable, similar to the `NUMBER_OF_CHANNELS` and `NUMBER_OF_NODES`. Works for now though.